apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -46,6 +49,58 @@ export function updateDefaultList(listData) {
     });
 }
 
+/* Regenerates the animation JSON list of the sprites in SpriteLab
+ */
+export function regenerateDefaultJSON() {
+  getDefaultList().then(json => {
+    let orderedKeys = [];
+    let propsByKey = {};
+    let parsed = spriteManifest;
+    let animations = parsed['metadata'];
+    for (let sprite of json.default_sprites) {
+      let animation_metadata = animations[sprite.key];
+      let props = {};
+      props['name'] = sprite.name;
+      props['sourceUrl'] = `https://studio.code.org${
+        animation_metadata['sourceUrl']
+      }`;
+      props['frameSize'] = animation_metadata['frameSize'];
+      props['frameCount'] = animation_metadata['frameCount'];
+      props['looping'] = animation_metadata['looping'];
+      props['frameDelay'] = animation_metadata['frameDelay'];
+      props['version'] = animation_metadata['version'];
+      props['categories'] = animation_metadata['categories'];
+      let key = createUuid();
+      orderedKeys.push(key);
+      propsByKey[key] = props;
+    }
+    let data = JSON.stringify({
+      orderedKeys: orderedKeys,
+      propsByKey: propsByKey
+    });
+    return fetch(`/api/v1/animation-library/default-spritelab-json`, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json'
+      },
+      body: data
```

Comment:
if this is the only place the `data` variable is used, we might want to set it directly here

```javascript
body: JSON.stringify({orderedKeys, propsByKey})
```